mopidy_ytmusic/playback.py

```diff
@@ -143,10 +143,25 @@ def _get_track(self, bId):
             # Use Youtube-DL's Info Extractor to decode the signature.
             if "signatureCipher" in playstr:
                 sc = parse_qs(playstr["signatureCipher"])
-                sig = self.PyTubeCipher.get_signature(
+
+                parsed_url = urlparse(sc["url"][0])
+                url_params = parse_qs(parsed_url.query)
+                url_params = {
+                    k: v[0] for k, v in url_params.items()
+                }
+
+                url_params["ratebypass"] = "yes"
+
+                url_params["sig"] = self.PyTubeCipher.get_signature(
                     ciphered_signature=sc["s"][0]
                 )
-                url = sc["url"][0] + "&sig=" + sig + "&ratebypass=yes"
+
+                if "n" in url_params:
+                    initial_n = url_params['n']
+                    discovered_n = self.PyTubeCipher.get_throttling(initial_n)
+                    url_params["n"] = discovered_n

```

Comment:
besides switching to `pytubefix`, which seems to have newer regexes for finding the cipher, this change seems to be quite important to get any playback - otherwise I was only getting forbidden URLs all the time